composer.json

```diff
@@ -20,12 +20,15 @@
         "dotkernel"
     ],
     "require": {
-        "php": "~7.4.0 || ~8.0.0 || ~8.1.0",
+        "php": "~8.1.0 || ~8.2.0",
         "laminas/laminas-servicemanager": "^3.10",
         "league/oauth2-facebook" : "^2.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.5"
+        "laminas/laminas-coding-standard": "^2.5",
+        "mikey179/vfsstream": "^1.6",
```

Comment:
I did not see this library being used anywhere in your code.
If it's not planned to be used later, please remove it.